### PR TITLE
Update the docs to better explain error handling with takeErrors()

### PR DESCRIPTION
**Pull request by samhagman**
- The docs still reference `endOnError()` in the error handling section, so I replaced that link with one to `takeErrors()`.
 - I also added an example of how to replicate the old functionality as described in the [deprecation of `endOnError()`.](https://github.com/rpominov/kefir/issues/150)
 - I also clarified that `takeErrors()` allows values to "just flow through".

### CODE REVIEW

File: docs-src/descriptions/errors.jade [hunk 1]
```diff
@@ -39,4 +39,33 @@ p.
...
+    > [stream.map.filter.takeErrors] <end:current>
+
+pre(title='events in time')
+  :escapehtml
+    foo: ---0---2---e---3---X
+                    -1
+
+    bar: -------4---eX
+                    -1
+div
```
Comment (rpominov) [inline docs-src/descriptions/errors.jade]: Please add a new line here.

File: docs-src/descriptions/one-source.jade [hunk 2]
```diff
@@ -147,7 +147,8 @@ div
 
 
 +descr-method('take-errors', 'takeErrors', 'obs.takeErrors(n)').
-  Same as #[a(href='#take') take] but for #[a(href='#about-errors') errors].
+  Emits the first #[b n] #[a(href='#about-errors') errors] from the
+  original observable, then ends.  Values just flow through.
```
Comment (rpominov) [inline docs-src/descriptions/one-source.jade]: Please remove the extra " " :smile:

**Comment by rpominov** [comment]
Makes sense. Sorry for the pedantic comments :smile:

**Comment by samhagman** [comment]
Don't worry, I can be just as pedantic haha :smiley:

**Comment by rpominov** [comment]
Great, thanks :+1: 

Will rebuild the website a bit later.

**Pull request merged**